### PR TITLE
Add utils section to create tfrecords from CVAT XML

### PR DESCRIPTION
**Pull request by idrissbellil**
Add utils section to convert CVAT XML format to tfrecords and the associated documentation. The script doesn't process `tracks` yet.

### CODE REVIEW

**Comment by nmanovic** [comment]
@idrissbellil , thank you for the contribution. Could you please help us to fix codacy issues in the patch?

File: utils/tfrecords/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+argparse>=1.1
+tensorflow
+pathlib
+Pillow
```
Comment (nmanovic) [inline utils/tfrecords/requirements.txt]: Probably it is better to fix version of libraries to be sure that the script will work in the future.

Comment (idrissbellil) [reply utils/tfrecords/requirements.txt]: @nmanovic thank you for the note, I will fix them shortly.

**Comment by idrissbellil** [comment]
@nmanovic Thanks for the note, I really don't know how to indent properly the list in the file [README.md](https://github.com/opencv/cvat/blob/develop/utils/README.md) in a way that Codacy won't show any warnings.

File: utils/tfrecords/converter.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+# Defining the main conversion function
+def create_tf_example(example, cvat_name, saved_dict, img_dir):
+  # Process one image data per run
+  height = int(example.attrib['height']) # Image height
+  width = int(example.attrib['width']) # Image width
+  filename = os.path.join(img_dir, example.attrib['name'])
+  filename = filename.encode('utf8')
+  with tf.gfile.GFile(filename,'rb') as fid:
+    encoded_jpg = fid.read()
```
Comment (nmanovic) [inline utils/tfrecords/converter.py]: What if the image directory contains png or bmp or mix of files?

File: utils/tfrecords/converter.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+pip3 install --user Cython
+pip3 install --user contextlib2
+pip3 install --user pillow
+pip3 install --user lxml
+pip3 install --user jupyter
+pip3 install --user matplotlib
+```
+```bash
+# clone and compile the cocoapi
+git clone https://github.com/cocodataset/cocoapi.git
```
Comment (nmanovic) [inline utils/tfrecords/converter.md]: How is cocoapi used?

Comment (idrissbellil) [reply utils/tfrecords/converter.md]: The cocoapi is being used in the compilation process of the object detection API which doesn't seem to compile without it. I don't know the details but It's part of the guide [INSTALL TENSORFLOW OBJECT DETECTION API](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/installation.md).

File: utils/tfrecords/converter.py (hunk 2 above)
Comment (idrissbellil) [reply utils/tfrecords/converter.py]: Following this guide [USING YOUR OWN DATASET](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/using_your_own_dataset.md), it seems like the `dataset_utils` does only support `.jpg` and `.png`, I will process these two cases, show a warning and leave if it does find some other image extension.

**Pull request closed without merging**